main.go

```diff
@@ -233,6 +251,33 @@ func fetchPages(team, since, until string) ([]*page, error) {
 	return pages, nil
 }
 
+func fetchTeamForPagerdutyIncident(client *pagerduty.Client, incidentId string) (string, error) {
+	alertsResp, err := client.ListIncidentAlerts(incidentId)
+	if err != nil {
+		return "", err
+	}
+
+	for _, a := range alertsResp.Alerts {
+		if details, ok := a.Body["details"]; ok {
```

Comment:
Is there a better/easier way to this sort of sequential transformation? e.g. something functional?

Reply:
Nope, but this many nested ifs should be avoided 😄 Looks like you could use a helper here.

Reply:
Done

Reply:
I unnested the ifs and moved to a helper